### DISCUSSION TITLE
Adding Prebuffer to videos

### DISCUSSION THREAD

**Discussion by candry7731** [jellyfin-archive/jellyfin-roku-legacy#595] [Ideas]
Looking to see if anyone has interest in adding prebuffer controls to videos when a user is on a details screen like, a movies details screen or episode detail screen. This would likely **improve** user experience. However could also **increase** transcoding resources.

This would speed up the launch of a video however we would be adding transcoding resources while prebuffering... 

The logic would be as follows:
When focus is on a movie details page the prebuffer starts. If the user plays the movie it loads and starts playing. If the user backs out of the details page to the main grid the prebuffer control stops, thus transcoding stops.

**Comment by jimdogx** [comment]
I can only speak for how we use Roku, but a lot of times we will click in to a movie just to read the details and see what it's about....then move on to the next one.  I'm more than fine with a little buffering in the beginning and think that's better than consuming resources spinning up transcoding etc.

Netflix, Disney, etc have huge wallets with great big servers and bandwidth, but Jellyfin can be run on anything down to a router or a Raspberry Pi 😄
